ThriftLibrary.cmake

```diff
@@ -166,13 +166,17 @@ endmacro()
 
 #
 # thrift_generate
+# Supports libary names that differ from the file name (two handle two libraries with the same filename on disk (in different folders))
 # This is used to codegen thrift files using the thrift compiler
 # Params:
-#   @file_name - The name of tge thrift file
+#   @file_name - The name of the library that is generated (without the language suffix)
```

Comment:
I think we should say that this is the name of the Thrift file without extension (and possibly say that it is also used to name targets if `TARGET_NAME_BASE` is not specified).